src/core_modules/capture-ui/internal/DateTimeInput/DateTimeDate.component.js

```diff
@@ -1,18 +1,19 @@
 // @flow
 import React from 'react';
 import { withFocusSaver } from '../../HOC/withFocusSaver';
-import { DateField } from '../../DateAndTimeFields/DateField/Date.component';
+import { withTextFieldFocusHandler } from '../TextInput/withFocusHandler';
 import { withShrinkLabel } from '../../HOC/withShrinkLabel';
-
+import { DateField } from '../../DateAndTimeFields/DateField/Date.component';
 
 function DateTimeDatePlain(props) {
     const { value, ...passOnProps } = props;
+
     return (
         <DateField
-            value={value || ''}
+            value={value}
             {...passOnProps}
         />
     );
 }
 
-export const DateTimeDate = withFocusSaver()(withShrinkLabel()(DateTimeDatePlain));
+export const DateTimeDate = withFocusSaver()(withShrinkLabel()(withTextFieldFocusHandler()(DateTimeDatePlain)));
```

Comment:
When manually typing in the Date part of the `DateTime` field, the placeholder covers the date. Can the placeholder be removed while typing like in the `Age` or `Date` fields? Thank you!Thanks!

<img width="776" alt="Screenshot 2024-10-31 at 08 57 38" src="https://github.com/user-attachments/assets/2b954c9d-3ccd-463b-88ae-49794ee81dd9">